tests/readers/test_csv_reader.py

```diff
@@ -141,11 +141,11 @@ def test_csv_with_comment():
         comment_line="This is a comment",
     )
 
-    df = s.to_pandas(pandas_read_csv_kwargs={"comment": "#"})
```

Comment:
We still need to add a test case that is explicitly using `pandas_read_csv_kwargs`. With this change there is none.

Reply:
added test: `test_csv_5`